content/hardware/03.nano/boards/nano-esp32/tutorials/debugging/debugging.md

```diff
@@ -47,23 +66,41 @@ To use the debugging feature on the Nano ESP32, you need to have the IDE 2.2.0 o
 
 If this is your first time debugging your code we recommend starting with the classic Blink example. It's a simple sketch but works great for understanding the basics of debugging.
 
-Before starting a debug session make sure to always do one of the following:
+Before starting a debug session you need to upload your sketch using one of the following methods:
+
+### Method 1
+
+- Connect a jumper cable between the **GND** and the **B1** pins and press the reset button **once**. The RGB LED will turn on with a green or blue color. 
 
-- Either connect a jumper cable between the **GND** and the **B1** pins and press the reset button **once**. The RGB LED will turn on with a green or blue color. Remove the jumper cable and you should see the RGB LED light up in a purple or yellow color. Then inside **Tools**, the board will be shown as a random ESP32 board.
+- Remove the jumper cable and you should see the RGB LED light up in a purple or yellow color. Note that inside **Tools**, the board will be shown as a random ESP32 board.
 
 ***This is because in this mode all ESP32 chips share the same identifier assigned to USB devices, therefore the IDE selects a random ESP32 board.***
 
-Continue to select **Tools** > **Board** > **Arduino Nano ESP32** as well as the correct **Port**.
+- Continue to select **Tools** > **Board** > **Arduino Nano ESP32** as well as the correct **Port**.
 
-After that select **Sketch** > **Upload Using Programmer**.
+![Select Port](./assets/selectPort.png)
 
-- Or double-tap the reset button. You will see the RGB LED fading slowly which means you correctly entered the recovery / Device Firmware Update (DFU) mode. You should see two ports in the drop-down menu, one showing a USB symbol and another one showing a cube.
+- After that select **Sketch** > **Upload Using Programmer**.
+
+### Method 2
+
+- Slowly double-tap the reset button to enter the recovery / Device Firmware Update (DFU) mode.
```

Comment:
I thought this was how we decided not to name these modes, 
- Double tap = Arduino bootloader mode, this is our software feature specific to Arduino boards 
- ROM Boot mode = Shorting B1, Hardware feature inherent to the ESP32 chip.

check the cheat sheet if this is unclear

Reply:
You are correct! This was taken from the developer's instructions and forgotten to change.